services/IoT/applications/libraries/python.md

```diff
@@ -743,7 +738,7 @@ The following code sample will return the device management information for a de
 ## Device diagnostic operations
 {: #device_diag_ops}
 
-Applications can use device diagnostic operations to clear logs, retrieve all or specific logs for a device, add log information, delete logs, clear error codes, get device error codes, and add an error codes.
+Applications can use device diagnostic operations to clear logs, retrieve all or specific logs for a device, add log information, delete logs, clear error codes, get device error codes, and add error codes.
```

Comment:
line 755
"The response model for this method contains the logId, message, severity, data, and timestamp."
are these parameters or are these items?
if they're items, then logId should be log ID.
if they're parameters, then add parameter after timestamp.